kscripts/generate_video.kts

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env kscript
 @file:DependsOn("com.github.ajalt:clikt:2.6.0")
+@file:DependsOn("com.google.code.gson:gson:2.8.5")
```

Comment:
https://twitter.com/JakeWharton/status/1265998249476993026 

Jokes aside, I try to avoid gson these days but works for me for a script that we touch only once a year.